plugins/src/webrtcsink/imp.rs

```diff
@@ -242,22 +245,45 @@ impl Default for State {
     }
 }
 
+fn make_converter_for_video_caps(caps: &gst::Caps) -> Result<gst::Element, Error> {
+    assert!(caps.size() == 1);
+
+    for feature in caps.features(0) {
+        if feature.contains(CUDA_MEMORY_FEATURE) {
+            return Ok(gst::parse_bin_from_description(
+                "cudaupload ! cudaconvert ! cudascale ! videorate drop-only=true",
```

Comment:
```suggestion
                "cudaupload ! cudaconvert ! cudascale ! videorate drop-only=true skip-to-first=true",
```

Is maybe wanted here. Was already wrong before then though :)

Reply:
Did you forget to update that one? :)